node/overseer/src/lib.rs

```diff
@@ -338,15 +342,34 @@ impl<M: Send + 'static> SubsystemContext for OverseerSubsystemContext<M> {
 		}).await.map_err(Into::into)
 	}
 
-	async fn send_message(&mut self, msg: AllMessages) -> SubsystemResult<()> {
-		self.tx.send(ToOverseer::SubsystemMessage(msg)).await.map_err(Into::into)
+	async fn send_message(&mut self, msg: AllMessages) {
+		self.send_logging_error(ToOverseer::SubsystemMessage(msg)).await
 	}
 
-	async fn send_messages<T>(&mut self, msgs: T) -> SubsystemResult<()>
+	async fn send_messages<T>(&mut self, msgs: T)
 		where T: IntoIterator<Item = AllMessages> + Send, T::IntoIter: Send
 	{
 		let mut msgs = stream::iter(msgs.into_iter().map(ToOverseer::SubsystemMessage).map(Ok));
-		self.tx.send_all(&mut msgs).await.map_err(Into::into)
+		if self.tx.send_all(&mut msgs).await.is_err() {
```

Comment:
Sending can only fail when the overseer has closed the channel. Do we really need to log this?

Reply:
This can be helpful for debugging purposes when running tests. Normally we should not see it fail when if the overseer sends `Conclude` signal in test environment. I can downgrade the log level to `debug` though.

Reply:
Perfect, ty